### PR TITLE
Group members according to activity_status

### PR DESCRIPTION
**Pull request by anuschka**
Addressing #164. It groups and displays kittens. I will group mentors above kittens.

### CODE REVIEW

File: web/templates/pages/members.html [hunk 1]
```diff
@@ -16,42 +16,47 @@
...
-					{% elif member.activity_status == 'ACTIVE' %}
-						<li class="active-cat">
-					{% elif member.activity_status == 'LAZY' %}
-						<li class="lazy-cat">
-					{% elif member.activity_status == 'SLEEPY' %}
-						<li class="sleepy-cat">
-					{% endif %}	
-
+				<div class="col-md-3 col-xs-6">
+					{% if member.user.is_active and member.activity_status == 'ACTIVE' %}
```
Comment (mojcas) [inline web/templates/pages/members.html]: Remember we have Active field on User as well, and those who are not active, are already filtered out in view, but on UserProfile we have an activity status. Here it seems now we are not displaying even Sleepy or Lazy Kittens? Why we have those statuses then? :)

Comment (anuschka) [inline web/templates/pages/members.html]: You are correct @mojcas. We could display mentors above everybody else
(active lazy and sleepy).

Comment (mojcas) [inline web/templates/pages/members.html]: Yep, it's good :)

File: web/templates/pages/members.html [hunk 2]
```diff
@@ -16,42 +16,82 @@
...
-				{% endif %}	
+				</div>
+			{% if forloop.last %}
+				</div><!-- closing row class when there are no more members in the list-->
+			{% else %}
+				{% if forloop.counter|divisibleby:"4" %}
+					</div><!-- closing row class after 4 memember in a row-->
+					<div class="row"><!--adding a new row class for the next line of members-->
+				{% endif %}
+			{% endif %}
```
Comment (ialja) [inline web/templates/pages/members.html]: This is getting hard to read. If you're closing a div, it's nice to have it indented the same way the div you're closing is.

**Pull request merged**